docs/asciidoc/antora.yml

```diff
@@ -1,11 +1,11 @@
 name: apoc
-version: 5.0
+version: 2025.0
```

Comment:
This is following how the 5.x series docs look, not sure if it is the best way, but keeps the root name at 2025 and then the actual release is correct. This is just to unblock the team publishing, so the extended team can make this better if they want :)